packages/polaris-viz/src/components/DonutChart/components/InnerValue/InnerValue.tsx

```diff
@@ -39,6 +44,10 @@ export function InnerValue({
     },
   });
 
+  useEffect(() => {
```

Comment:
Does this need to be a useEffect?

Could we not do `const fontSize = diameter * SCALING_FACTOR`?